exporter/clickhousetracesexporter/migrations/000014_add_usage_explorer_mv.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS signoz_traces.usage_explorer (
+  timestamp DateTime64(9) CODEC(DoubleDelta, LZ4),
+  service_name LowCardinality(String) CODEC(ZSTD(1)),
+  count UInt64 CODEC(T64, ZSTD(1))
+) ENGINE SummingMergeTree()
+PARTITION BY toDate(timestamp)
+ORDER BY (timestamp, service_name);
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS signoz_traces.usage_explorer_mv
+TO signoz_traces.usage_explorer
+AS SELECT
+  toStartOfHour(timestamp) as timestamp,
+  serviceName as service_name,
+  count() as count
+FROM signoz_traces.signoz_index_v2
+GROUP BY toStartOfHour(timestamp) as timestamp, serviceName;
```

Comment:
should this be just `GROUP BY timestamp, serviceName`

Reply:
Updated